flang/test/Lower/empty-omp-parallel.f90

```diff
@@ -0,0 +1,36 @@
+! This test checks lowering of OpenMP parallel Directive.
```

Comment:
Can we have a directory called OpenMP which contains all the OpenMP tests?

Reply:
Sure, as we move forward. It's better to have one.